omnipaxos/src/omni_paxos.rs

```diff
@@ -36,8 +34,7 @@ pub struct OmniPaxosConfig {
     pub peers: Vec<u64>,
     pub buffer_size: usize,
     pub skip_prepare_use_leader: Option<Ballot>,
-    pub leader_quorum_size: Option<usize>,
-    pub append_quorum_size: Option<usize>,
+    pub flexible_quorum: Option<(usize, usize)>,
```

Comment:
On a second thought, it is probably more clear to have a custom struct `FlexibleQuorum` that the user faces:
```
pub struct FlexibleQuorum {
        read_quorum_size: usize,
        write_quorum_size: usize,
}
```

```suggestion
    pub flexible_quorum: Option<FlexibleQuorum>,
```

Reply:
our Quorum enum can then use it as well `Quorum::Flexible(FlexibleQuorum)`